### ISSUE TITLE
Tab margins cause test logs to emit on a single line

### ISSUE THREAD

**Issue by zbeckman** [etorreborre/specs2#587]
If you write your code with tabs for indents, as we do:
```
def is = s2"""
	Given a Kafka log in the b03 package
	When writing to the log ${canWrite}
	Then the message can be read back ${canRead}
	"""
```
Then when specs2 runs the test, the logs are emitted on a single line:
```
[info] LogWorks
[info]    Given a Kafka log in the b03 package+ When writing to the log* Then the message can be read back PENDING
```

**Comment by etorreborre** [comment]
I published a fix in `3.9.4-20170815061805-0e62822`. It is pretty simple in the sense that it just replaces the tabs by 2 spaces before slicing the text into specification Fragments.

Note that you will get a better display by indenting your spec like
```
s2"""
Given a Kafka log in the b03 package
  When writing to the log ${canWrite}
  Then the message can be read back ${canRead}
"""
```
Even when using tabs. Because in that case `Given ...` is interpreted as a stand-alone piece of text. Otherwise when you write
```
s2"""
  Given a Kafka log in the b03 package
  When writing to the log ${canWrite}
  Then the message can be read back ${canRead}
"""
```
The whole `Given a Kafka log in the b03 package  When writing to the log` is interpreted as a multi-line description for an example.

**Comment by zbeckman** [comment]
Outstanding! 👍
